api/app/controllers/spree/api/base_controller.rb

```diff
@@ -93,7 +117,7 @@ def error_during_processing(exception)
         error_notifier.call(exception, self) if error_notifier
 
         render text: { exception: exception.message }.to_json,
-          status: 422 and return
+          status: 422 && return
```

Comment:
This won't run as you think it would. It should be:

```ruby
render(text: { exception: exception.message }.to_json, status: 422) && return
```

What you're actually doing:

```ruby
render(text: { exception: exception.message }.to_json, status: 422 && return0
```